src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
let's destructure the `pesron`, so it'll be more convenient to use the props
`const { someValue, someOtherValue } = person`